infra/core/ai/cognitiveservices.bicep

```diff
@@ -34,8 +34,8 @@ resource account 'Microsoft.CognitiveServices/accounts@2023-10-01-preview' = {
   properties: {
     customSubDomainName: customSubDomainName
     publicNetworkAccess: publicNetworkAccess
-    // Document Intelligence (FormRecognizer) does not support bypass in network acls
-    networkAcls: kind == 'FormRecognizer' ? networkAcls : networkAclsWithBypass
+    // Some services do not support bypass in network acls
+    networkAcls: (kind == 'FormRecognizer' || kind == 'ComputerVision') ? networkAcls : networkAclsWithBypass
```

Comment:
This change is needed after the Vnet change. I'm surprised noones filed an issue about it, as you get a deployment error when deploying a vision branch.

Reply:
Yeah I have gotten a report about it already. Can you please make this a separate PR?

Reply:
Sure, https://github.com/Azure-Samples/azure-search-openai-demo/pull/1657/files